Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mdbook-mermaid"
-version = "0.13.0"
+version = "0.14.0"
```

Comment:
Please remove this. It's handled by tooling.

Reply:
Done